Versionize/Git/RepositoryExtensions.cs

```diff
@@ -30,9 +32,11 @@ public static bool VersionTagsExists(this Repository repository, SemanticVersion
         return repository.Tags.Any(tag => tag.FriendlyName.Equals(tagName));
     }
 
-    public static bool IsSemanticVersionTag(this Tag tag)
+    public static bool IsSemanticVersionTag(this Tag tag, bool omitTagVersionPrefix = false)
```

Comment:
@cabauman: This method is unused, not even in the tests. Since it belongs to the public API, I didn't dare to remove it. However, you may want to keep that in mind and remove it at a later point in time, except if you know other projects that sit on top of versionize and use this API.